easybuild/easyconfigs/c/cdat/cdat-8.2.1-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,132 @@
+easyblock = 'PythonBundle'
+name = 'cdat'
+version = '8.2.1'
+versionsuffix = "-Python-%(pyver)s"
+
+homepage = 'https://github.com/CDAT/cdat'
+description = """CDAT is a powerful and complete front-end to a rich set of 
+visual-data exploration and analysis capabilities well suited for data analysis problems."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'pic': True}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('Qt5', '5.14.1'),
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.0.4'),
+    ('libxml2', '2.9.10'),
+    ('libxslt', '1.1.34'),
+    ('SQLite', '3.31.1'),
+    ('HDF5', '1.10.6'),
+    ('bzip2', '1.0.8'),
+    ('OpenSSL', '1.1.1e'),
+    ('Tk', '8.6.10'),
+    ('expat', '2.2.9'),
+    ('libffi', '3.3'),
+    ('Mesa', '20.0.2'),
+    ('DB', '18.1.32'),
+    ('gdbm', '1.18.1'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+    ('netCDF-C++4', '4.3.1'),
+    ('libcdms', '3.1.2'),
+    ('unidata', '2.8', versionsuffix),
+    ('VTK', '8.2.0', versionsuffix),
+]
+
+# todo installscript na esg
```

Comment:
Is this still a TODO?

Reply:
Nope, "esg is dead" say the official docs